### PR TITLE
Fix a few small bugs/confusions in JS 4

### PR DESCRIPTION
**Pull request by samsworldofno**

### CODE REVIEW

File: js/lesson4/tutorial.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ The following code allows you to listen for a keypress on the input field, and t
 
 ```js
 $(document).ready(function(){
-  $(document).on('keypress', '#username', function(e){
+  $(document).on('keypress', '#username', function(){
```
Comment (jkbits1) [inline js/lesson4/tutorial.md]: 'e' was wrong, but shouldn't it be 'event' ?

File: js/lesson4/tutorial.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ We will need to pass the username to GitHub, so we need to extract it from the i
...
```
Comment (jkbits1) [inline js/lesson4/tutorial.md]: Same as above - 'e' was wrong, but shouldn't it be 'event' as the param ?

**Comment by jkbits1** [comment]
These look good, I've added a couple of minor line comments (about the event params)

File: js/lesson4/tutorial.md (hunk 1 above)
Comment (samsworldofno) [inline js/lesson4/tutorial.md]: It works with `(e)` and then `event` called... leading me to believe that `event` is something native in JS? I'll whip up a test to check.

Comment (jkbits1) [inline js/lesson4/tutorial.md]: Ah, maybe. I just saw a student type it last night and assumed it was a typo, so we changed it before debugging it.

Comment (samsworldofno) [inline js/lesson4/tutorial.md]: Yep I've just tested it. The `e` is not needed - `event` is given to you somehow.

Comment (jkbits1) [inline js/lesson4/tutorial.md]: Ok, great. Didn't know that!

**Comment by jkbits1** [comment]
Great, merged

**Pull request merged**